examples/benchmarks/bubblebench/common/src/test/common.test.ts

```diff
@@ -3,4 +3,5 @@
  * Licensed under the MIT License.
  */
 
+// eslint-disable-next-line unicorn/no-empty-file
 describe("Bubblebench", () => {});
```

Comment:
Should we just delete this file if it has no tests? If these are the only tests in the package, and strictly exist to make the mocha script pass, we should add a comment to that effect.